docs/specs/glossary.md

```diff
@@ -0,0 +1,18 @@
+# Glossary
+
+local coordinates
+:   The local coordinate system is the coordinate system of the current
+    group or layer, with the X coordinate increasing towards the right
+    and the Y coordinate increasing towards the bottom.
```

Comment:
Does this mean that [0,0] local coordinate would be in the top left of the current group/layer?

Reply:
You can have negative coordinates, [0, 0] corresponds to the top-left corner of the frame (assuming no transforms)

Reply:
I mentioned this